src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -710,6 +710,13 @@ public class AgentKnobs
             new EnvironmentKnobSource("DISABLE_RESOURCE_UTILIZATION_WARNINGS"),
             new BuiltInDefaultKnobSource("false"));
 
+        public static readonly Knob DisableResourceMonitorDebugOutput = new Knob(
```

Comment:
This should be EnableResourceMonitorDebugOutput, default off.  As it is, the functionality will be enabled on agent upgrade and add risk at time of upgrading agent.  Risk should be shifted to when enabling the feature flag.

Reply:
Done. Thanks!